sway-lsp/src/handlers/notification.rs

```diff
@@ -0,0 +1,102 @@
+//! This module is responsible for implementing handlers for Language Server
+//! Protocol. This module specifically handles notifications.
```

Comment:
In case it's not obvious what LSP notifications are:

```suggestion
//! Protocol. This module specifically handles async messages from the client.
```

Reply:
Ended up going with:

```
//! This module is responsible for implementing handlers for Language Server
//! Protocol. This module specifically handles notification messages sent by the Client.
```